packages/framework/presence/src/systemWorkspace.ts

```diff
@@ -114,6 +115,7 @@ class SystemWorkspaceImpl implements PresenceStatesInternal, SystemWorkspace {
 		public readonly events: IEmitter<
 			Pick<PresenceEvents, "attendeeJoined" | "attendeeDisconnected">
 		>,
+		private readonly runtime: IEphemeralRuntime,
 	) {
```

Comment:
Only needs audience, right? Then let's limit exposure to just audience instead of entire ephemeral runtime.